cloud/installation/prerequisites.md

```diff
@@ -7,7 +7,7 @@ title: Prerequisites
 
 ## OS
 
-The poller must be installed on a dedicated fresh Alma Linux/RHEL/Oracle Linux 8 or 9, or Debian 11 server. Debian 12 is also supported for pollers from version 24.04. See our knowledge base article [How to install Linux to host Centreon software](https://thewatch.centreon.com/product-how-to-21/how-to-install-linux-to-host-centreon-software-3759).
+The poller must be installed on a dedicated fresh Alma Linux/RHEL/Oracle Linux 8 or 9, or Debian 11 server. Debian 12 is also supported for pollers from version 24.04. See our knowledge base articles [How to install Linux to host Centreon software](https://thewatch.centreon.com/product-how-to-21/how-to-install-linux-to-host-centreon-software-3759) and [OS updates & security](https://thewatch.centreon.com/product-how-to-21/os-updates-security-3136).
```

Comment:
Cloud pollers should move to 24.10 once we update the Cloud to 24.10, and therefore Debian 11 won't be supported there (something to keep in mind for a change in the next few weeks).

Reply:
I opened a ticket for this. Thanks!